docs/docs/migration/migration-from-v7-to-v8.md

```diff
@@ -172,6 +172,31 @@ Perform the following changes:
     }
     ```
 
+### Rename `Menu` and related components to `MainNavigation` in `@comet/admin`
```

Comment:
Should the code-migration script be mentioned here? 🤔 

https://github.com/vivid-planet/comet-upgrade/pull/45

Reply:
Good point - thank you! Added here: 63a4d0763cd94a742525326d72bb403a954f3c50

Reply:
Ah, sorry, I thought these upgrade-scripts had to be run manually and that's why I thought the cli-command should be mentioned in the docs. 

This isn't the case however, as (I think) your upgrade-script would be run automatically when upgrading to `v8` with `npx @comet/upgrade@latest v8`. So this mention in the docs isn't actually necessary.